lib/client-assets.php

```diff
@@ -66,6 +66,28 @@ function gutenberg_get_script_polyfill( $tests ) {
 	return $polyfill;
 }
 
+/**
+ * Registers the main TinyMCE scripts.
+ */
+function register_tinymce_scripts() {
+	global $tinymce_version, $concatenate_scripts, $compress_scripts;
+	if ( ! isset( $concatenate_scripts ) ) {
+		script_concat_settings();
+	}
+	$suffix     = SCRIPT_DEBUG ? '' : '.min';
+	$compressed = $compress_scripts && $concatenate_scripts && isset( $_SERVER['HTTP_ACCEPT_ENCODING'] )
+		&& false !== stripos( $_SERVER['HTTP_ACCEPT_ENCODING'], 'gzip' );
+	// Load tinymce.js when running from /src, otherwise load wp-tinymce.js.gz (in production) or
+	// tinymce.min.js (when SCRIPT_DEBUG is true).
+	$mce_suffix = false !== strpos( get_bloginfo( 'version' ), '-src' ) ? '' : '.min';
```

Comment:
I think there are similar pieces of code in Gutenberg that grab the WP version string. Maybe it would be good to create a separate function for it. We should probably include the version file so we're sure it's not modified by plugins. See `gutenberg_pre_init`.

Reply:
> We should probably include the version file so we're sure it's not modified by plugins

Why? I mean if Core uses this function `get_bloginfo( 'version' )` why not just use this function in `gutenberg_pre_init` as well?

Reply:
> so we're sure it's not modified by plugins

Reply:
> so we're sure it's not modified by plugins

I guess if plugins modifies this, the classic editor would already break right?

Reply:
I'm not sure why? If this is the same code as core then it's fine I guess :)